From 1c5eae4b26be051da763aa5ff5626d30eb0c09c7 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@freescale.com>
Date: Fri, 24 Apr 2015 13:31:13 -0500
Subject: [PATCH] MLK-10753 mxc IPUv3: common: Increase IPU priority for 2nd
 capture channel

Set the IPU channel priority to high for the second CSI
capture channel.

Signed-off-by: Oliver Brown <oliver.brown@freescale.com>

Conflicts:

	drivers/mxc/ipu3/ipu_common.c
---
 drivers/mxc/ipu3/ipu_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/ipu3/ipu_common.c b/drivers/mxc/ipu3/ipu_common.c
index faad068..9073c33 100644
--- a/drivers/mxc/ipu3/ipu_common.c
+++ b/drivers/mxc/ipu3/ipu_common.c
@@ -654,7 +654,7 @@ static int ipu_probe(struct platform_device *pdev)
 	}
 
 	/* Set sync refresh channels and CSI->mem channel as high priority */
-	ipu_idmac_write(ipu, 0x18800001L, IDMAC_CHA_PRI(0));
+	ipu_idmac_write(ipu, 0x18800003L, IDMAC_CHA_PRI(0));
 
 	/* Enable error interrupts by default */
 	ipu_cm_write(ipu, 0xFFFFFFFF, IPU_INT_CTRL(5));
-- 
1.8.0

